### PR TITLE
PalitanX Integration

### PR DESCRIPTION
**Pull request by estherleongym**
- Added Pairs, Tickers, OrderBook, Trades, and updated README for PalitanX
- Related to issue #839 
- [ X ] I have added Specs
- [ X ] (If implementing Market Ticker) I have verified that the `volume` refers to BASE
- [ X ] (If implementing Market Ticker) I have verified that the `base` and `target` is assigned correctly

### CODE REVIEW

File: lib/cryptoexchange/exchanges/palitanx/services/market.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          ticker.base      = market_pair.base
+          ticker.target    = market_pair.target
+          ticker.market    = Palitanx::Market::NAME
+          ticker.last      = NumericHelper.to_d(output['Last'].to_f)
+          ticker.high      = NumericHelper.to_d(output['High'].to_f)
+          ticker.low       = NumericHelper.to_d(output['Low'].to_f)
+          ticker.bid       = NumericHelper.to_d(output['Bid'].to_f)
+          ticker.ask       = NumericHelper.to_d(output['Ask'].to_f)
+          ticker.volume    = NumericHelper.to_d(output['BaseVolume'].to_f)
+          ticker.timestamp = Time.now.to_i
```
Comment (wongy91) [inline lib/cryptoexchange/exchanges/palitanx/services/market.rb]: returns `nil` instead

**Comment by estherleongym** [comment]
@wongy91 done and pushed

**Comment by wongy91** [comment]
@estherleongym  im getting error calling the pairs.

```
irb(main):005:0> client.pairs 'palitanx'
Cryptoexchange::HttpResponseError: {:response=>#<HTTP::Response/1.1 503 Service Unavailable {"Date"=>"Wed, 29 Aug 2018 04:54:35 GMT", "Content-Type"=>"text/html; charset=UTF-8", "Transfer-Encoding"=>"chunked", "Connection"=>"close", "Set-Cookie"=>"__cfduid=dc9cd735d06e4412ea8b4bf4ca44676681535518475; expires=Thu, 29-Aug-19 04:54:35 GMT; path=/; domain=.palitanx.com; HttpOnly", "X-Frame-Options"=>"SAMEORIGIN", "Cache-Control"=>"no-cache", "Expect-Ct"=>"max-age=604800, report-uri=\"https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\"", "Server"=>"cloudflare", "Cf-Ray"=>"451c6ba788f92cc3-KUL"}>}
```

**Comment by estherleongym** [comment]
@wongy91 I created another PR (#987 ) for this, client.pairs('palitanx') works now.

**Pull request closed without merging**